_includes/components/header--basic.html

```diff
@@ -40,9 +40,15 @@
               {% endfor %}
             </ul>
             {% else %}
-            <a class="usa-nav-link{% if _section.href == page.permalink %} usa-current{% endif %}" href="{% if _section.external %}{{ _section.href }}{% else %}{{ _section.href | relative_url }}{% endif %}">
-              <span>{{ _section.text }}</span>
-            </a>
+              {% if _section.type == "Button" or _section.type == "button"%}
```

Comment:
Is there a better way to do this than the OR statement to capture both `button` and `Button` ?

Reply:
One approach could be to use `button: true` rather than `type: button` in navigation.yml